### PR TITLE
use packit to build both packages

### PR DESCRIPTION
**Pull request by TomasTomecek**

### CODE REVIEW

**Comment by praiskup** [comment]
Very nice!

**Comment by TomasTomecek** [comment]
kudos @lachmanfrantisek for implementing it in Packit that you can have separate jobs with specific identifiers

**Comment by praiskup** [comment]
Thanks!

File: .packit.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  identifier: mock
+  metadata:
+    targets:
+    - fedora-rawhide
+  actions:
+    create-archive:
+    - bash -c "cd mock/ && tito build --tgz --test -o ."
+    - bash -c "ls -1t ./mock/*.tar.gz | head -n 1"
+    get-current-version:
+    - bash -c "git describe --match mock-[0-9]* --abbrev=0 HEAD | egrep -o [0-9]+\.[0-9]+"
```
Comment (xsuchy) [inline .packit.yaml:17]: I would probably argue for 
`git tag -l 'mock-[0-9]*' | sort | tail -n1`

Comment (xsuchy) [reply .packit.yaml:17]: Hmm, no, that do something different. Ignore this my comment.

**Pull request merged**